src/components/DynamicFormContainer/DynamicForm/AttachmentDropzone/FilesInfo/FilesInfo.test.tsx

```diff
@@ -17,12 +17,155 @@ describe("filesInfo", () => {
       />
     );
     expect(screen.queryByTestId("upload-file-0")).not.toBeNull();
-    expect(screen.queryByTestId("attachment-icon-0")).not.toBeNull();
+    expect(screen.queryByTestId("attachment-icon-application/pdf")).not.toBeNull();
```

Comment:
u shouldn't be needing to suffix mimetype yeah? can use `attachment-icon` is fine